tests/suites/test_suite_psa_crypto.function

```diff
@@ -2348,6 +2358,20 @@ void copy_success( int source_usage_arg,
         ASSERT_COMPARE( material->x, material->len,
                         export_buffer, length );
     }
+
+    /* Convert wildcard algorithm to exercisable algorithm */
```

Comment:
Given that #4141 doesn't have any initial review yet, and we need this one to land sooner rather than later, I object to making this wait for #4141.

I can either make the change you requested and move the logic into `exercise_mac_key` and `exercise_aead_key` now in this PR, or I can leave it to you to move the logic as part of #4141 afterwards. Preference?

Reply:
This is a maintenability issue, not a correctness issue, so I'm fine with handling it in a follow-up once both #4067 and #4141 is merged.

However, there's already a merge conflict with #4141, and I'm not ok with delaying #4141 until #4067 is merged. Our intended schedule for #4141 is reviews posted Tuesday, rework posted Wednesday this week, so it might land this week if no further rework is necessary. So it's likely that this PR will need a rebase or a merge with #4141 anyway.

Reply:
I'm fixing your feedback *right now*, meaning this should be good to go tonight or early tomorrow.

The intended schedule for this one was also to get everything defined, decided and reviewed over the course of a week. It's been almost 4 now. Forgive me if I sound jaded, but given the historic track record I'm skeptical about #4141 actually landing anytime this week.

Reply:
Turns out #4141 was ready today, so I just merged it. I understand you were opposed to waiting for #4141, in case it took longer to be ready than we expected, which is a fair concern indeed, but since #4141 is ready now, I think it resolves this concern.

Obviously merging #4141 means this PR will need to address the conflict, but doing it the other way round would also require resolving conflicts, and the longer we wait, the likely we are to see conflicts with other PRs as well, so I thought it was best to merge #4141 as soon as it was ready.